### ISSUE TITLE
Attribute Error in MoveToMapPokemon

### ISSUE THREAD

**Issue by Naiters** [PokemonGoF/PokemonGo-Bot#4661]
### Behavior
Bot crashes - MoveToMapPokemon issue.

### Your FULL config.json (remove your username, password, gmapkey and any other private info)
Same as map.example.

### Output when issue occurred
Traceback (most recent call last):
  File "pokecli.py", line 703, in <module>
    main()
  File "pokecli.py", line 118, in main
    bot.tick()
  File "C:\Python27\PokemonGo-Bot\pokemongo_bot\__init__.py", line 568, in tick
    if worker.work() == WorkerResult.RUNNING:
  File "C:\Python27\PokemonGo-Bot\pokemongo_bot\cell_workers\move_to_map_pokemon.py", line 292, in work
    pokemon_list = self.get_pokemon_from_social()
  File "C:\Python27\PokemonGo-Bot\pokemongo_bot\cell_workers\move_to_map_pokemon.py", line 105, in get_pokemon_from_social
    if not self.bot.mqtt_pokemon_list or len(self.bot.mqtt_pokemon_list) <= 0:
AttributeError: 'PokemonGoBot' object has no attribute 'mqtt_pokemon_list'

I am running **latest** dev.

**Comment by sohje** [comment]
@Naiters I dunno why this is merged, but you can open cell_workers/move_to_map_pokemon.py
Comment out 292 line and uncomment 294.
```python
        # pokemon_list = self.get_pokemon_from_social()
        #Temp works as it, need add more configuration
        pokemon_list = self.get_pokemon_from_map()
```

**Comment by mjmadsen** [comment]
Merged a fix #4680
